schema/defs.yaml

```diff
@@ -179,7 +179,7 @@ description: Common schema definitions shared by all themes
         The complete lists of allowed values can be found at:
         https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
       type: string
-      pattern: "^[a-zA-Z]{2,3}(-[a-zA-Z]{4})?(-[a-zA-Z]{2})?$"
+      pattern: "^(?:(?:[A-Za-z]{2,3}(?:-[A-Za-z]{3}){0,3}?)|(?:[A-Za-z]{4,8}))(?:-[A-Za-z]{4})?(?:-[A-Za-z]{2}|[0-9]{3})?(?:-(?:[A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))?(?:-[A-WY-Za-wy-z0-9](?:-[A-Za-z0-9]{2,8})+)?$"
```

Comment:
Can you add a description of this regex?

Reply:
Yup, next time I have a minute I'll update it.

Reply:
Updated with documentation. Could you take a look, and resolve your comment if you feel it's addressed?